pkg/iac-providers/cft/v1/load-file.go

```diff
@@ -48,6 +49,18 @@ func (a *CFTV1) LoadIacFile(absFilePath string) (allResourcesConfig output.AllRe
 	}
 	allResourcesConfig = make(map[string][]output.ResourceConfig)
 	for _, doc := range iacDocuments {
+
+		// replacing yaml data as the default yaml.v3 removes
+		// intrinsic tags for cloudformation templates
+		// (!Ref, !Fn::<> etc are removed and resolved to a string
+		// which disables parameter resolution by goformation)
+		templateData, err := ioutil.ReadFile(absFilePath)
```

Comment:
I think `ioutil.ReadFile(absFilePath)` should happen only for yaml. This place is applicable for both json and yaml.

Reply:
Good catch, thanks